src/Header/Header.js

```diff
@@ -1,9 +1,11 @@
 // Dependency Imports
 import React, { useEffect, useContext, useState } from "react";
 import clsx from "clsx";
+import ApiStatus from "../APIStatus/APIStatus";
 import { makeStyles, useTheme } from "@material-ui/core/styles";
 import MenuIcon from "@material-ui/icons/Menu";
 import { RiSensorFill, RiTimeLine } from "react-icons/all";
+import ErrorIcon from "@material-ui/icons/Error";
```

Comment:
Icon imports are part of `@material-ui/core`. I recommend using the already existing imports(Line 20)  for icons instead of importing them separately. 
[Tree Shaking](https://webpack.js.org/guides/tree-shaking/)